bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -144,10 +144,19 @@ export default function Whiteboard(props) {
     return hasShapeAccess;
   }
 
+  const isValidShapeType = (shape) => {
+    const invalidTypes = ['image'];

```

Comment:
Need to add the `video` type aswell

Reply:
added in https://github.com/bigbluebutton/bigbluebutton/pull/16477/commits/419e83ecd40ef70cff83eced803f4ee5594459f4